API.yaml

```diff
@@ -1552,6 +1561,30 @@ components:
       properties:
         parameters:
           $ref: '#/components/schemas/RequestedParameters'
+    RequestedFilterParameters:
+      required:
+      - is_full_search
+      - filter_expressions_map
+      type: object
+      properties:
+        is_full_search:
+          type: boolean
+        filter_expressions_map:
+          type: object
+          properties:
+            key:
+              type: string
+              description: where the key 1 is DIMMED and 4 is EXCLUDED
```

Comment:
Idem

Reply:
Hi MatthewKhouzam, 

I did following bhufmann's recommendation above 'Map<Integer, Collection<String>> filterExpressionsMap;'. 
As I understood, the frontend is supposed to rematch based on 'DIMMED = 1 << 0' or 'EXCLUDE = 1 << 2'